### ISSUE TITLE
[Question] How to receive the last telemetry that triggered the alarm dynamically

### ISSUE THREAD

**Issue by nowherex** [thingsboard/thingsboard#4892]
**Component**
 * Generic

**Description**
Hi, I'm trying to send notifications by email when I receive a notification created from a device profile, in the data received in the rule chains (debug) I don't receive the telemetry value that triggered the alarm, 
I saw in the documentation a exemple that the alarm is sent in "details":, but I only get "null" as the example below, would anyone know a way to get the value of the last telemetry that triggered the alarm dynamically?

Data example from alarm:

```
{
    "tenantId": {
        "entityType": "TENANT",
        "id": "efacdf10-d70e-11eb-99cb-1520a2e6ced5"
    },
    "type": "High Temperature",
    "originator": {
        "entityType": "DEVICE",
        "id": "cafd3e00-dc84-11eb-9d0e-1f8899a6f9b3"
    },
    "severity": "MAJOR",
    "status": "ACTIVE_UNACK",
    "startTs": 1626136353258,
    "endTs": 1626136353258,
    "ackTs": 0,
    "clearTs": 0,
    "details": null,
    "propagate": true,
    "propagateRelationTypes": [],
    "id": {
        "entityType": "ALARM",
        "id": "d08e5310-e371-11eb-af07-9b635ba9a793"
    },
    "createdTime": 1626136353473,
    "name": "High Temperature"
}
```

**Environment**
 * OS:  Linux Mint 20.2
 * ThingsBoard: Thingsboard.cloud
 * Browser: Google Chrome

**Comment by dhaval2017** [comment]
use originator telemetry rule node to fetch latest value
![image](https://user-images.githubusercontent.com/33371842/125732308-0043b579-9a0f-457a-be3f-23a464cb49f4.png)

**Comment by LeoMorgan113** [comment]
@nowherex 

I agree with the @dhaval2017 suggestion of using 'Originator telemetry'(follow the https://thingsboard.io/docs/pe/user-guide/rule-engine-2-0/enrichment-nodes/#originator-telemetry) 
This node adds Message Originator telemetry values from particular time range that was selected in node configuration to the Message Metadata.

**Comment by volodymyr-babak** [comment]
@nowherex 
you can add values of the telemetry that are used in alarm rule condition to the alarm details, and take them from 'details.data' field of the alarm object:
![image](https://user-images.githubusercontent.com/3466101/151805510-47da21f5-c70b-4a99-8122-ef677dc73c3f.png)
![image](https://user-images.githubusercontent.com/3466101/151805598-133a5627-297c-4c46-b386-1f4192754133.png)

But please make sure that this telemetry is used in alarm rule condition, in other case (if telemetry is present inside TB msg, but not used in alarm condition) substitution will not work.
